src/aics-image-viewer/components/App/index.tsx

```diff
@@ -15,23 +15,14 @@ import {
   VolumeFileFormat,
 } from "@aics/volume-viewer";
 
-import {
-  AppProps,
-  ShowControls,
-  GlobalViewerSettings,
-  ViewerSettingUpdater,
-  ViewerSettingChangeHandlers,
-  UseImageEffectType,
-} from "./types";
+import type { AppProps, ShowControls, UseImageEffectType } from "./types";
+import type { ViewerState, ChannelState } from "../ViewerStateProvider/types";
```

Comment:
probably should either put a newline after these imports or else group them with the ones below here that don't come from "shared".